apidoc/Titanium/UI/Clipboard/Clipboard.yml

```diff
@@ -153,6 +153,63 @@ methods:
     osver: {ios: {min: "10.0"}}
     platforms: [iphone, ipad]
 
+  - name: remove
+    summary: Removes the clipboard.
+    since: "6.1.0"
+    platforms: [iphone, ipad]
+
+properties:
+  - name: name
+    summary: Create a new named clipboard.
+    description: Create a new clipboard with a given name. See the example on how to use this.
+    parameters:
+      - name: name
+        summary: name of clipboard.
+        type: String
+    since: "6.1.0"
+    platforms: [iphone, ipad]
+    availability: creation
+
+  - name: unique
+    summary: Create a new clipboard identified by a unique system-generated name.
+    description: Create a new clipboard identified by a unique system-generated name. See the example 
+    on how to use this.
+    parameters:
+      - name: unique
+        summary: value of unique.
+        type: Boolean
+    since: "6.1.0"
+    platforms: [iphone, ipad]
+    availability: creation
+
+  - name: allowCreation
+    summary: Create a clipboard identified by name if it doesn,t exist.
+    description: |
+        If create is false, it will return the existing named clipboard but will not create new one.
+        If create is true, it will return the existing clipboard (if exists) otherwise will create new 
+        clipboard with given name.
+    parameters:
+      - name: create
+        summary: value of create.
+        type: Boolean
+    since: "6.1.0"
+    platforms: [iphone, ipad]
+    availability: creation
+
+  - name: persistent
+    summary: A Boolean value that indicates whether the clipboard is persistent.
+    description: |
+        When a clipboard is persistent, it continues to exist past app terminations and across system 
+        reboots. Clipboard that are not persistent only last until the owning (creating) app quits.
+    parameters:
+      - name: persistent
+        summary: value of persistent.
+        type: Boolean
+        default: false in case of named clipboard otherwise true.
+    since: "6.1.0"
+    osver: {ios: {max: "9.x.x"}}
```

Comment:
Actually, we should leave this whole property out, as we should not include deprecated iOS-API's in new Ti-API's:
> Persistent named pasteboards are deprecated in iOS 10. The system sets the persistence of all pasteboards automatically, with named pasteboards marked as nonpersistent and the systemwide general pasteboard marked as persistent. If you try to set the setPersistent(_:) property on a pasteboard, Xcode issues a deprecation warning.
Instead of persistent named pasteboards, used shared containers. For information on shared containers, read about the containerURL(forSecurityApplicationGroupIdentifier:) method in FileManager, and read about the com.apple.security.application-groups entitlement key in Adding an App to an App Group in Entitlement Key Reference.

Thoughts?